src/helpers/auth.js

```diff
@@ -0,0 +1,49 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const secret = process.env.SECRET;
+
+const Authenticate = {
+  /**
+   * @description - this method encodes a token
+   * @param {object} user
+   * @return {string} token
+   */
+  encode(user) {
+    const token = jwt.sign({ user }, secret, { expiresIn: '72h' });
```

Comment:
Kindly make secret and expiresIn dynamic variables i.e function caller should be able to set those parameters

Reply:
it will make sense for expiresIn but not for secret